test/metabase/api/card_test.clj

```diff
@@ -1370,17 +1369,17 @@
     (public-settings/enable-query-caching true)
     (testing "card ttl only"
       (mt/with-temp* [Card [card {:cache_ttl 1337}]]
-        (is (= 1337 (:cache-ttl (card-api/query-for-card card {} {} {}))))))
+        (is (= (* 3600 1337) (:cache-ttl (card-api/query-for-card card {} {} {}))))))
     (testing "multiple ttl, dash wins"
       (mt/with-temp* [Database [db {:cache_ttl 1337}]
                       Dashboard [dash {:cache_ttl 1338}]
                       Card [card {:database_id (u/the-id db)}]]
-        (is (= 1338 (:cache-ttl (card-api/query-for-card card {} {} {} {:dashboard-id (u/the-id dash)}))))))
+        (is (= (* 3600 1338) (:cache-ttl (card-api/query-for-card card {} {} {} {:dashboard-id (u/the-id dash)}))))))
     (testing "multiple ttl, db wins"
       (mt/with-temp* [Database [db {:cache_ttl 1337}]
                       Dashboard [dash]
                       Card [card {:database_id (u/the-id db)}]]
-        (is (= 1337 (:cache-ttl (card-api/query-for-card card {} {} {} {:dashboard-id (u/the-id dash)}))))))
+        (is (= (* 3600 1337) (:cache-ttl (card-api/query-for-card card {} {} {} {:dashboard-id (u/the-id dash)}))))))
```

Comment:
Do you reckon a `hrs->secs` fn is overkill here?  Just noticing this happens four times in the PR.

Reply:
I vacillated over it for 30 minutes and decided 3600 was obvious enough. If we ever touch this again and this confuses anyone then it's time for it but right now I think YAGNI is the case